src/partials/notice-banner.hbs

```diff
@@ -9,6 +9,7 @@
       page.componentVersion.asciidoc.attributes.olderVersion))
     }}
     <div class="notice-banner">
+      <button class="notice-banner-close-button" title="Close the notice banner">&times;</button>
```

Comment:
I know I'm being picky on this, but we should move this to a labels file, too.  

I'm sure we have other strings that aren't properly translated, but we should do this properly anyway.